superset-frontend/plugins/plugin-chart-echarts/src/components/Echart.tsx

```diff
@@ -92,11 +92,10 @@ function Echart(
     previousSelection.current = currentSelection;
   }, [currentSelection]);
 
-  useEffect(() => {
-    if (chartRef.current) {
-      chartRef.current.resize({ width, height });
-    }
-  }, [width, height]);
+  // Ensure that each render and resize is idempotent
+  if (chartRef.current) {
+    chartRef.current.resize({ width, height });
```

Comment:
will this trigger an expensive operation on the chart? Cause this will run on each render now.
Could useLayoutEffect work?

Reply:
Yeah. You are right. Thanks for the good suggestion! @diegomedina248